CHANGES.rst

```diff
@@ -61,6 +61,8 @@ API Changes
 
 - ``astropy.io.ascii``
 
+  - ASCII writers now accept an 'overwrite' argument. [#1233]
```

Comment:
Add: 
```
The default behavior is changed so that a warning will be 
issued when overwriting an existing file unless ``overwrite=True``.
In a future version this will be changed from a warning to an
exception to prevent accidentally overwriting a file.
```